### PR TITLE
operations

### LINKED ISSUE #26: operations

### PR DESCRIPTION
**Pull request by theganyo**
closes #26

### CODE REVIEW

File: quota/manager.go [hunk 1]
```diff
@@ -53,7 +53,7 @@ Dispatch basically works like this:
 // A Manager tracks multiple Apigee quotas
 type Manager interface {
 	Start()
-	Apply(auth *auth.Context, p *product.APIProduct, args Args) (*Result, error)
+	Apply(auth *auth.Context, o product.AuthorizedOperation, args Args) (*Result, error)
```
Comment (rockspore) [inline quota/manager.go:56]: Should we use `ao` here? And why wasn't a pointer used here like before?

Comment (rockspore) [reply quota/manager.go:56]: Oh I see that this structure is much smaller than a API product.

File: product/manager_test.go [hunk 2]
```diff
@@ -170,8 +173,24 @@ func TestManagerPolling(t *testing.T) {
 	if pp1 == pp2 {
 		t.Errorf("number of products should have incremented")
 	}
+
+	ac := &auth.Context{
```
Comment (rockspore) [inline product/manager_test.go:177]: The name `auth` is used in quota pkg instead of `ac`. Could we make it consistent?

Comment (theganyo) [reply product/manager_test.go:177]: If it bothers you, sure. 😄

Comment (rockspore) [reply product/manager_test.go:177]: It normally doesn't but I happened to see it :)

Comment (theganyo) [reply product/manager_test.go]: It was everywhere, but it's all nice now.

**Review by rockspore — APPROVED** [review]

**Pull request merged**